units/patch_intent.go

```diff
@@ -175,12 +175,23 @@ func (j *patchIntentProcessor) Run() {
 	}
 
 	if j.Intent.ShouldFinalizePatch() {
-		if _, err := model.FinalizePatch(patchDoc, githubOauthToken); err != nil {
+		if _, err := model.FinalizePatch(patchDoc, intentToRequester(j.Intent.GetType()), githubOauthToken); err != nil {
 			j.AddError(err)
 		}
 	}
 }
 
+func intentToRequester(r string) string {
```

Comment:
could this be a method on a patch intent, or a function that takes the intent interface type?

Reply:
done